packages/fxa-auth-server/lib/senders/emails/templates/verifyAccountChange/en.ftl

```diff
@@ -0,0 +1,19 @@
+# Variables:
+# $code (String) - The verification code
+verifyAccountChange-subject = Use { $code } to change your account
+# Variables:
+# $expirationTime (Number) - Represents the expiration time in minutes
+verifyAccountChange-preview = { $expirationTime: ->
```

Comment:
I missed that the syntax is incorrect here (the colon is not part of Fluent variants).